packages/ui/internal/localized-fonts/localized-fonts.js

```diff
@@ -53,13 +53,19 @@ const buildFontSet = function (fontName, fonts, strLang, bitDefault) {
 	return strOut;
 };
 
-function fontGenerator (locale) {
```

Comment:
Enact CLI directly runs `@enact/moonstone/MoonstoneDecorator/fontGenerator` directly to prerender the localized font style block in the localized prerendered index html files. That directly calls `@enact/ui/internal/localized-fonts`. So if I'm understanding this change, Enact CLI would be updated to support passing `localeChanged` argument?

Reply:
Pasting response from slack: CLI shouldn't have to be updated, since failing to pass a 2nd arg (`localeChanged`) - while resulting in a truthy value in the added check (!undefined = true) - will still fail the 1st check, resulting in executing the code underneath since the `<style>` tag doesn't exist. The 2nd arg is only for runtime use - to force an update to the `<style>` block when it exists.